src/panels/lovelace/cards/hui-glance-card.ts

```diff
@@ -91,6 +85,10 @@ class HuiGlanceCard extends HassLocalizeLitMixin(LitElement)
       (conf) => conf.entity in states
     );
 
+    if(this.config.theme) {
```

Comment:
We will need to call `applyThemesOnElement` regardless if a theme is picked or not, or else we won't clean up when config.theme is not set. Do this by setting theme to `"default"`